scripts/update-rules.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const recommendedRulesFile = require('../lib/config/recommended');
+const octaneRulesFile = require('../lib/config/octane');
+const stylisticRulesFile = require('../lib/config/stylistic');
+const root = require('../lib/rules');
+
+const pathReadme = path.resolve(__dirname, '../README.md');
+const readmeContent = fs.readFileSync(pathReadme, 'utf8');
+const tablePlaceholder = /<!--RULES_TABLE_START-->[\S\s]*<!--RULES_TABLE_END-->/;
+
+// Config/preset emojis.
+const STAR = ':white_check_mark:';
+const OCTANE = ':car:';
+const STYLISTIC = ':dress:';
+
+// Generate rule table contents.
+let rulesTableContent = Object.keys(root)
+  .map((name) => {
+    // Check which configs this rule is part of.
+    const isRecommended = Object.prototype.hasOwnProperty.call(recommendedRulesFile.rules, name);
+    const isOctane = Object.prototype.hasOwnProperty.call(octaneRulesFile.rules, name);
+    const isStylistic = Object.prototype.hasOwnProperty.call(stylisticRulesFile.rules, name);
+    const mark = `${isRecommended ? STAR : ''}${isOctane ? OCTANE : ''}${
+      isStylistic ? STYLISTIC : ''
+    }`;
+
+    const link = `[${name}](./docs/rule/${name}.md)`;
+
+    return `| ${mark} | ${link} |`;
+  })
+  .join('\n');
+
+fs.writeFileSync(
+  pathReadme,
+  readmeContent.replace(
+    tablePlaceholder,
+    `<!--RULES_TABLE_START-->\n\n|    | Rule ID |\n|:---|:--------|\n${rulesTableContent}\n<!--RULES_TABLE_END-->`
```

Comment:
If we have two `\n` after the table start, we should also have two `\n` before the table end.

Reply:
Fixed.